src/urls.ts

```diff
@@ -22,27 +22,49 @@ export const getWorkUrl = ({
   return workUrl;
 };
 
-export const getAsShortUrl = ({ url }: { url: string }) => url.replace(/archiveofourown/, 'ao3');
+export const getAsShortUrl = ({ url }: { url: string }) =>
+  url.replace(/archiveofourown/, "ao3");
 
 export const getUserProfileUrl = ({ username }: { username: string }) =>
   `https://archiveofourown.org/users/${encodeURI(username)}/profile`;
 
-const tokenReplacerObject = {
-  '/': '*s*',
-  '&': '*a*',
-  '.': '*d*',
-  '#': '*h*',
-  '?': '*q*'
-}
-const tokensToEscape = ['/', '?', '.']
-const replaceEscapableTokens = (c:string) => tokensToEscape.includes(c) ? `\\${c}` : c;
-const tagUrlReplaceChars = new RegExp(`(${Object.keys(tokenReplacerObject).map(replaceEscapableTokens).join('|')})`, 'g')///(\/|\.|&|#|\?)/g;
+const TOKEN_REPLACEMENTS_MAP = {
```

Comment:
I follow the "global constants should be screaming snake case" convention pretty closely. Definitely need to get some type of linter in here rather than be telling people though, so my bad.

For "Object" at the end of the name, I prefer avoiding names that describe the type, especially since TS will give you the hint anyway. I could also go for `TOKEN_REPLACEMENTS`, but I feel like `MAP` is a good middle ground since it's what it's used for, not the type.